app/src/main/java/org/mifos/mobile/ui/about/AboutUsViewModel.kt

```diff
@@ -0,0 +1,68 @@
+package org.mifos.mobile.ui.about
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.mifos.mobile.MifosSelfServiceApp.Companion.context
+import org.mifos.mobile.R
+import org.mifos.mobile.models.AboutUsItem
+import org.mifos.mobile.ui.enums.AboutUsListItemId
+import java.util.*
+import javax.inject.Inject
+
+@HiltViewModel
+class AboutUsViewModel @Inject constructor() : ViewModel() {
+
+    private val _aboutUsItemEvent = MutableLiveData<AboutUsListItemId>()
+    val aboutUsItemEvent: LiveData<AboutUsListItemId> get() = _aboutUsItemEvent
+
+    private val currentYear = Calendar.getInstance().get(Calendar.YEAR)
+    private val copyrightText =
+        context?.getString(R.string.copy_right_mifos)
+            ?.replace("%1\$s", currentYear.toString())
+
+    val aboutUsItems: List<AboutUsItem> = listOf(
+        AboutUsItem(
+            context?.getString(R.string.app_version_text),
+            null,
```

Comment:
You don't need to pass `null` value here, the default value is already `null`

Reply:
done